tests/ui/regular-output-format-fail/expected

```diff
@@ -1,4 +1,2 @@
-Check 1: main.assertion.2
 Description: "assertion failed: 1 + 1 == 3"
-Description: "pointer to dead object"
```

Comment:
Is this check no longer produced with `--crate-type lib`?

Reply:
correct